qiskit/qpy/binary_io/circuits.py

```diff
@@ -587,7 +626,7 @@ def _write_instruction(file_obj, instruction, custom_operations, index_map, use_
             not hasattr(library, gate_class_name)
             and not hasattr(circuit_mod, gate_class_name)
             and not hasattr(controlflow, gate_class_name)
-            and gate_class_name != "Clifford"
+            and gate_class_name not in ["Clifford", "AnnotatedOperation"]
```

Comment:
Not that it matters with 2 elements, but I assume this will continue to grow and a set lookup will be more efficient:

```suggestion
            and gate_class_name not in {"Clifford", "AnnotatedOperation"}
```